A3-channel-tracing.md

```diff
@@ -21,48 +21,35 @@ This design uses the same terminology discussed in the [channelz](A14-channelz.m
 In addition, the following terminology will be used:
 
 1.  a "Trace event" is an interesting thing that happens to a channel. Examples include things like creation, address resolution, subchannel creation, connectivity state changes.
-2. "Trace data" is all of the data that is contained for one channel. This includes a list of _Trace events_, as well as metadata like the timestamp at which the channel was created.
-3.  a "Tracing object" is an in-memory data structure responsible for holding the _Trace data_ for a single channel.
+2.  a "Channel trace" is a data structure responsible for holding all data for a single channel. This includes a list of _Trace events_, as well as metadata like the timestamp at which the channel was created. The _Channel trace_ also holds a list of _Channel trace_ object of its child channels (if it has any).
 
 ## Abstract
 
-The document proposes adding a dedicated _Tracing object_ for every channel and subchannel. The _Tracing object_ will record important events in the life of a channel, like address resolution, subchannel creation, channel state changes, etc. The _Trace data_ from this _Tracing object_ will be made available through the [channelz service](A14-channelz.md).
+The document proposes adding a dedicated _Channel trace_ for every channel and subchannel. The _Channel trace_ will record important events in the life of a channel, like address resolution, subchannel creation, channel state changes, etc. The data from this _Channel trace_ will be made available through the [channelz service](A14-channelz.md).
```

Comment:
In the "Proposal" section below, we should have some detailed guidance about what sorts of events should and should not be added to the channel trace.  In particular, I think we should state that no events associated with individual calls should be added, because that will very quickly fill up the buffer.

Reply:
I added an example of trace from a healthy channel